example/App.js

```diff
@@ -1,9 +1,14 @@
 import Expo from 'expo';
 import React from 'react';
-import { StyleSheet, Text, TouchableOpacity, View } from 'react-native';
+import { StyleSheet, Text, View, ScrollView } from 'react-native';
 import { Entypo } from '@expo/vector-icons';
 import { ActionSheetProvider, connectActionSheet } from '@expo/react-native-action-sheet';
 
+const saveIcon = require('./img/baseline_save_black_24dp.png');
+const shareIcon = require('./img/baseline_share_black_24dp.png');
+const cancelIcon = require('./img/baseline_cancel_black_24dp.png');
+const deleteIcon = require('./img/baseline_delete_black_24dp.png');
```

Comment:
can we possibly utilize the icons that are shipped with Expo? We're already using Entypo, we could utilize those or use SimpleLineIcons for iOS and Material for Android

Reply:
done like dinner.